src/main/resources/assets/sbmsheepmetal/lang/de_DE.json

```diff
@@ -0,0 +1,59 @@
+{
+    "_comment1": "Misc",
+    "entity.sbmsheepmetal:sheep.metal": "Metallwollenschaf",
+
+    "_comment2": "Item names",
+    "item.sbmsheepmetal.wool_item_copper": "Kupferwolle",
+    "item.sbmsheepmetal.wool_item_tin": "Zinnwolle",
+    "item.sbmsheepmetal.wool_item_lead": "Bleiwolle",
+    "item.sbmsheepmetal.wool_item_silver": "Silberwolle",
+    "item.sbmsheepmetal.wool_item_uranium": "Uranwolle",
+    "item.sbmsheepmetal.wool_item_brass": "Messingwolle",
+    "item.sbmsheepmetal.wool_item_bronze": "Bronzewolle",
+    "item.sbmsheepmetal.wool_item_steel": "Stahlwolle",
+    "item.sbmsheepmetal.wool_item_electrum": "Elektrumwolle",
+    "item.sbmsheepmetal.wool_item_nickel": "Nickelwolle",
+    "item.sbmsheepmetal.wool_item_aluminum": "Aluminiumwolle",
+    "item.sbmsheepmetal.wool_item_zinc": "Zinkwolle",
+    "item.sbmsheepmetal.wool_item_platinum": "Platinwolle",
+    "item.sbmsheepmetal.wool_item_titanium": "Titanwolle",
+    "item.sbmsheepmetal.wool_item_gold": "Goldwolle",
+    "item.sbmsheepmetal.wool_item_iron": "Eisenwolle",
+    "item.sbmsheepmetal.metal_sheep_spawn_egg": "Erschaffe Metallwollenschaf",
```

Comment:
Please keep this in line with the vanilla spawn egg translations.

Reply:
Oh, seems like they changed the translation. Will fix ASAP

Reply:
Done